web/src/main/java/org/jboss/cdi/tck/tests/implementation/builtin/metadata/session/YoghurtInterceptor.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.cdi.tck.tests.implementation.builtin.metadata.session;
+
+import java.io.Serializable;
+
+import jakarta.enterprise.inject.Intercepted;
+import jakarta.enterprise.inject.spi.Bean;
+import jakarta.enterprise.inject.spi.Interceptor;
+import jakarta.inject.Inject;
+import jakarta.interceptor.AroundInvoke;
+import jakarta.interceptor.InvocationContext;
+import org.jboss.cdi.tck.tests.full.decorators.implementation.builtin.metadata.Frozen;
+
+@SuppressWarnings("serial")
+@jakarta.interceptor.Interceptor
+@Frozen
+public class YoghurtInterceptor implements Serializable {
+
+    @Inject
+    private Bean<org.jboss.cdi.tck.tests.full.decorators.implementation.builtin.metadata.YoghurtInterceptor> bean;
```

Comment:
Imports and FQCN of some classes in this interceptor are wrong. It should reference classes in this package instead.